packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/DashboardWidget.kt

```diff
@@ -25,21 +27,18 @@ import javax.persistence.GenerationType
 import javax.persistence.Id
 import javax.persistence.JoinColumn
 import javax.persistence.ManyToOne
-import javax.persistence.Table
 import javax.persistence.Temporal
 import javax.persistence.TemporalType
 import javax.persistence.Transient
 
 @Entity
 @Include(type = "dashboardWidgets")
-@Table(name = "dashboard_widgets")
-@SharePermission
-@CreatePermission(expression = "is an author OR is an editor")
-@UpdatePermission(expression = "is an author now OR is an editor now")
-@DeletePermission(expression = "is an author now OR is an editor now")
+@CreatePermission(expression = "$IS_AUTHOR OR $IS_EDITOR")
+@UpdatePermission(expression = "$IS_AUTHOR OR $IS_EDITOR")
+@DeletePermission(expression = "$IS_AUTHOR OR $IS_EDITOR")
 class DashboardWidget : HasAuthor, HasEditors {
-    @get:Id
-    @get:GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
A while back we had a discussion about the best way to do hashcode/equals for Elide models.  We should probably follow that here.

Reply:
Yes, that did come to mind when refactoring, especially when using `Set`s.

This technique: https://vladmihalcea.com/how-to-implement-equals-and-hashcode-using-the-jpa-entity-identifier/

Created an issue: https://github.com/yahoo/navi/issues/865